infra/modules/service/main.tf

```diff
@@ -319,3 +319,111 @@ resource "aws_security_group" "app" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 }
+
+###########################
+## Database Configuration ##
+###########################
+resource "aws_rds_cluster" "postgresql" {
+  cluster_identifier = var.service_name
+  engine             = "aurora-postgresql"
+  engine_mode        = "provisioned"
+  database_name      = replace("${var.service_name}", "-", "_")
+  master_username    = "app_usr"
+  master_password    = aws_ssm_parameter.random_db_password.value
+  storage_encrypted  = true
+  # checkov:skip=CKV_AWS_128:IAM Auth will be added with the `wic-prp-eng` role created in PRP-74 https://wicmtdp.atlassian.net/browse/PRP-74
+  # checkov:skip=CKV_AWS_162:IAM Auth will be added with the `wic-prp-eng` role created in PRP-74 https://wicmtdp.atlassian.net/browse/PRP-74
+  # iam_database_authentication_enabled = true
+  deletion_protection = true
+
+
+  serverlessv2_scaling_configuration {
+    max_capacity = 1.0
+    min_capacity = 0.5
+  }
+
+}
+
+resource "aws_rds_cluster_instance" "postgresql-cluster" {
+  cluster_identifier         = aws_rds_cluster.postgresql.id
+  instance_class             = "db.serverless"
+  engine                     = aws_rds_cluster.postgresql.engine
+  engine_version             = aws_rds_cluster.postgresql.engine_version
+  auto_minor_version_upgrade = true
+  monitoring_role_arn        = aws_iam_role.rds_enhanced_monitoring.arn
+  monitoring_interval        = 30
+}
+
+resource "random_password" "random_db_password" {
+  length           = 48
+  special          = true
+  min_special      = 6
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+resource "aws_ssm_parameter" "random_db_password" {
+  name  = "/metadata/db/admin-password"
+  type  = "SecureString"
+  value = random_password.random_db_password.result
+}
+
+resource "aws_backup_plan" "postgresql" {
+  name = "${var.service_name}_backup_plan"
+
+  rule {
+    rule_name         = "${var.service_name}_backup_rule"
+    target_vault_name = "${var.service_name}-vault"
+    schedule          = "cron(0 12 ? * SUN *)"
+  }
+}
+# checkov:skip=CKV2_AWS_8:TODO add backup selection plan using tags
```

Comment:
Looks like this checkov is still failing. Including a backup selection as part of this PR should probably fix that